src/daemon/protocols/cdp.c

```diff
@@ -201,7 +201,38 @@ cdp_send(struct lldpd *global,
 	      POKE_END_CDP_TLV))
 		goto toobig;
 
-#ifdef ENABLE_LLDPMED
+#ifdef ENABLE_DOT3
+	if ((version >= 2) &&
+	    (port->p_power.powertype != LLDP_DOT3_POWER_8023AT_OFF) &&
+	    (port->p_power.devicetype == LLDP_DOT3_POWER_PD) &&
+	    (port->p_power.requested > 0) &&
+	    (port->p_power.requested <= 655)) {
+		if (port->p_power.requested != port->p_power.allocated) {
+			port->p_cdp_power.request_id++;
+			log_debug("cdp", "requested: %d, allocated:%d", port->p_power.requested, port->p_power.allocated);
+		}
+		u_int16_t consumption = port->p_power.allocated ? port->p_power.allocated : 154;
```

Comment:
What is the magic value 154? Maybe use a constant?

Reply:
Done